ndcube/ndcube.py

```diff
@@ -267,15 +279,8 @@ def dimensions(self):
 
     @property
     def world_axis_physical_types(self):
-        """
```

Comment:
You can't delete the the docstring here until we can rely on astropy/astropy#7166 as it currently dosen't inherit for properties.